README.md

```diff
@@ -56,37 +139,145 @@ cd ./server/server
 rushx dev
 ```
 
-After server is started, it will accept any incoming request for existing workspaces.
+After server is started, it accepts any incoming request for existing workspaces.
+
+## Development
+
+### Project Directory structure
+This project is setup to have multiple packages and services in the same repository (mono-repository) as such the 
+directory structure is quite important. Below, we briefly described, maybe not all, but the most important 
+directories of the project.
+
+```
+PLATFORM
+├─ LICENSE
+├─ README.md
+├─ CONTRIBUTING.md
+├─ package.json
+├─ rush.json
+├─ common - contains general configurtations for the project
+│  ├─ changes
+│  ├─ config
+│  ├─ git-hooks
+│  └─ scripts
+├─ dev - contains local development server
+│  ├─ platform-rig
+│  ├─ prod - frontend part of the project
+│  ├─ server - backend part of the project
+│  └─ ui-build
+├─ models - contains definitions for domain modals
+│  ├─ all
+│  ├─ calendar
+│  ├─ chunter
+│  ├─ core
+│  ├─ dev
+│  ├─ fsm
+│  ├─ meeting
+│  ├─ recruting
+│  ├─ task
+│  └─ workbench
+├─ packages - contains code shared among client and server, except packages/platform which defines plugin architecture and client-side only. 
+│  ├─ core
+│  ├─ model
+│  ├─ platform
+│  ├─ query
+│  ├─ richeditor
+│  ├─ router
+│  ├─ rpc
+│  ├─ status
+│  ├─ text
+│  ├─ theme
+│  ├─ ui
+│  └─ webrtco
+├─ plugins - contains client-side code packaged in form of Platform Plugins.
+│  └─ ...
+├─ products - contains examples of business aplications
+│  └─ hr-build
+├─ server - contains server-side code of the Platform
+│  ├─ accounts
+│  ├─ client
+│  ├─ elastic
+│  ├─ mongo
+│  ├─ server
+│  ├─ services
+│  ├─ tools
+│  ├─ workspace
+│  └─ workspaces
+└─ tests - contains automated tests for the Platform code
+   └─ server
+```
+
+ 
+### How to run dev server
```

Comment:
You can add information about run dev(demo) site without server, auth and any data storage(use memDB).  just `cd ./dev/prod` and `rushx dev`